themes/dg-theme/assets/css/main.scss

```
@@ -7,6 +7,11 @@ $screenheight-short: 900px;
 a, a:hover, a:visited, a:active {
   color: inherit;
   text-decoration: none;
+  white-space: pre-wrap; /* CSS3 */    
+  white-space: -moz-pre-wrap; /* Mozilla, since 1999 */
+  white-space: -pre-wrap; /* Opera 4-6 */    
+  white-space: -o-pre-wrap; /* Opera 7 */    
+  word-wrap: break-word; /* Internet Explorer 5.5+ */
```

We should go even one step further and just use `word-wrap: anywhere` here, on very small viewports (<800px) the links are still breaking responsiveness.
```suggestion
  word-wrap: anywhere; /* Internet Explorer 5.5+ */
```